O21.Game/Engine/GameRules.fs

```diff
@@ -14,6 +15,14 @@ let LevelHeight = 400
 [<Literal>]
 let TicksPerSecond = 10.0
 
+let mutable Config : Config =
```

Comment:
I'd prefer not to use global mutable bindings only to pass some data to scenes. Please use State for that.

Reply:
Thanks for review!